cases_high_level_simulation/test_3d_unit_position_based_bc.cpp/CMakeLists.txt

```diff
@@ -20,16 +20,15 @@ foreach(STL_FILE ${FILES_STL})
 endforeach()
 
 aux_source_directory(. DIR_SRCS)
-ADD_EXECUTABLE(${PROJECT_NAME} ${EXECUTABLE_OUTPUT_PATH}  ${DIR_SRCS})
```

Comment:
These changes are required for running the test in windows system.

Reply:
I put it back